app/src/main/kotlin/at/bitfire/davdroid/resource/LocalCalendarStore.kt

```diff
@@ -106,6 +110,11 @@ class LocalCalendarStore @Inject constructor(
         return values
     }
 
+    override fun delete(localCollection: LocalCalendar) {
```

Comment:
In the other data stores this method was moved to the end of the class. Here it's moved away from the end of the class ?